File: SpringBootCache/src/main/java/com/example/java/SpringBootCacheApplication.java
package com.example.java;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;

@SpringBootApplication
@EnableCaching
public class SpringBootCacheApplication {

	public static void main(String[] args) {
		SpringApplication.run(SpringBootCacheApplication.class, args);
	}
}

File: SpringBootCache/src/main/kotlin/com/example/kotlin/SpringBootCacheApplication.kt
package com.example.kotlin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching

@SpringBootApplication
@EnableCaching
class SpringBootCacheApplication

fun main(args: Array<String>) {
    runApplication<SpringBootCacheApplication>(*args)
}